spec/acceptance/steps/links_indexing_steps.rb

```diff
@@ -0,0 +1,114 @@
+step 'I create a draft content item' do
+  @content_id = SecureRandom.uuid
+  @base_path = "/" + SecureRandom.hex
+
+  payload = {
+    title: 'Hallo',
+    base_path: @base_path,
+    schema_name: 'detailed_guide',
+    document_type: 'detailed_guide',
+    publishing_app: 'end-to-end-publishing-test',
+    rendering_app: 'end-to-end-publishing-test',
+    locale: 'en',
+    routes: [
+      { path: @base_path, type: 'exact' }
+    ]
+  }
+
+  Services.publishing_api.put_content(@content_id, payload)
+
+  sleep 0.1
```

Comment:
Correct.

I've used a thing like `wait_until` to check if rummager contains the document here:

https://github.com/alphagov/govuk-publishing-pipeline-tests/blob/ff747d76c41d3584aaef9c693125d3d748fd724c/spec/acceptance/steps/links_indexing_steps.rb#L56-L66

I'll try to extract some things to use it elsewhere too.